packages/main/src/StepInput.hbs

```diff
@@ -0,0 +1,77 @@
+<div
+	id="{{_id}}"
+	class="ui5-step-input-root"
+	style="{{styles}}"
+	@keydown="{{_onkeydown}}"
+	@focusin="{{_onfocusin}}"
+	@focusout="{{_onfocusout}}"
+>
+	<!-- Decrement Icon -->
+	<div
+		class="ui5-step-icon ui5-step-dec"
+		title="{{decIconTitle}}"
+	>
+		<ui5-icon
+			id="{{_id}}-dec"

```

Comment:
you should enable show-tooltip for the icon

Reply:
Done